CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # New Relic Ruby Agent Release Notes
 
+## dev
+
+- **Feature: Add support for Trilogy**
```

Comment:
```suggestion
- **Feature: Add support for Trilogy database adapter**
```
I really associate the phrase "database adapter" when I think of the role Trilogy's playing within a Rails application. I'd like to see it worked somewhere into the entry. Maybe the title's a good spot?